Source/CombatExtended/CombatExtended/Jobs/Utils/JobGiverUtils_Reload.cs

```diff
@@ -165,6 +165,7 @@ public static Thing FindBestAmmo(Pawn pawn, Building_AutoloaderCE AutoLoader)
         {
             var ammoComp = AutoLoader.CompAmmoUser;
             AmmoDef requestedAmmo = ammoComp.SelectedAmmo;
+            Log.Message(requestedAmmo.defName);
```

Comment:
Is there some reason to be spamming the log here?  At the least it should go behind an `#if DEBUG` check.